src/assets/toolkit/styles/vendor/wordpress.css

```diff
@@ -0,0 +1,92 @@
+/* Wordpress generated classes */
```

Comment:
Can we add the URL to the codex CSS page to these comments for future generations?